FrontEnd/src/components/Contact/Contact.jsx

```diff
@@ -1,28 +1,201 @@
+                reset();
+            }
+        } catch (error) {
+            setModalMessage('Щось пішло не так! Будь ласка, спробуйте ще раз');
+        } finally {
+            setShowModal(true);
+            setLoading(false);
+        }
+    };
+
+    const handleCancel = () => {
+        reset();
+    };
+
+    const closeModal = () => {
+        setShowModal(false);
+    };
+
+
+    return (
+        <div className={styles['contact_container']}>
+            <div className={styles['contact__link_container']}>
+                <LinkContainer />
+                <img
+                    className={styles['contact__img1']}
+                    src={`${process.env.REACT_APP_PUBLIC_URL}/img/dots_12x10.png`}
+                    alt="dots_12x10.png"
+                />
+            </div>
+            <div className={styles['contact__text_container']}>
+                <h2 className={styles['contact__title']}>Зворотній зв&apos;язок</h2>
+                <form onSubmit={handleSubmit(onSubmit)} className={styles['contact__form']} noValidate>
+                    <div className={styles['contact__field']}>
+                        <label className={styles['contact__label']} htmlFor="email">Email:</label>
+                        <input className={styles['contact__input']}
+                            id="email"
+                            type="email"
+                            placeholder="Ваша пошта"
+                            {...register('email', {
+                                required: 'Обов’язкове поле',
+                                pattern: {
+                                    value: EMAIL_PATTERN,
+                                    message: 'Електронна пошта не відповідає вимогам',
+                                }
+                            })}
+                        />
+                        {errors.email && <p className={styles['contact__error']}>{errors.email.message}</p>}
+                    </div>
+                    <div className={styles['contact__field']}>
+                        <label className={styles['contact__label']} htmlFor="message">Категорія:</label>
+                        <Controller
+                            name="category"
+                            control={control}
+                            rules={{
+                                required: 'Будь ласка, оберіть тип повідомлення',
+                            }}
+                            render={({ field, fieldState: { error } }) =>
+                                <Space direction="vertical" style={{ width: '100%', gap: '0px', }}>
+                                    <Select
+                                        placeholder="Оберіть категорію"
+                                        {...field}
+                                        style={{
+                                        width: 400,
+                                        padding: '0px',
+                                        }}
```

Comment:
nit: It doesn't seem to be properly formatted

Reply:
fixed, thank you